### DISCUSSION TITLE
changedetection.io | configure variables

### DISCUSSION THREAD

**Discussion by smtdev** [tteck/Proxmox#948] [Q&A]
Hi,

I would like to know how to configure environment variables as Docker does (what I want to test is the `PLAYWRIGHT_DRIVER_URL` variable). I guess there is a `.conf` file but I am not able to find it.

Any help would be appreciated.

Thanks!

**Comment by smtdev** [comment]
@tteck can you please help me with this? Where can I play with the env variables?

Thank you.

**Reply by ka2er** [reply]
see maybe it would fit your needs ? #1349
